### PR TITLE
v0.11.0rc1

### PR DESCRIPTION
**Pull request by rustyrussell**
Finally!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,164 @@ All notable changes to this project will be documented in this file.
...
+ - JSON-RPC: listincoming showed incoming_capacity_msat field 1000 times actual value. ([#4913])
+ - Options: Respect --always-use-proxy AND --disable-dns when parsing wireaddresses to listen on. ([#4829])
+ - lightningd: remove slow memory leak in DEVELOPER builds. ([#4931])
+ - JSON-RPC: `paystatus` entries no longer have two identical `amount_msat` entries. ([#4911])
+ - We really do allow providing multiple addresses of the same type. ([#4902])
+
+
+### EXPERIMENTAL
+
+ - Fixex `experimental-websocket` intermittent read errors ([#5090])
```
Comment (endothermicdev) [inline CHANGELOG.md:91]: "Fixed" (here and line 95)

Comment (vincenzopalazzo) [reply CHANGELOG.md:91]: damnt autogen :)

**Review by vincenzopalazzo — APPROVED** [review]
ACK https://github.com/ElementsProject/lightning/pull/5155/commits/c3fb1501dfaa14260c681df45f83788a04d264d4

**Comment by jb55** [comment]
ocd nit:

```patch
From 8ff049a5be4f94ef763086d7b31e5f61354331bd Mon Sep 17 00:00:00 2001
From: William Casarin <jb55@jb55.com>
Date: Mon, 4 Apr 2022 12:49:47 -0700
Subject: [PATCH clightning 1/2] changelog: sort pr links

---
 CHANGELOG.md | 96 ++++++++++++++++++++++++++--------------------------
 1 file changed, 48 insertions(+), 48 deletions(-)

diff --git a/CHANGELOG.md b/CHANGELOG.md
index 54d7565f4..a3d4d858f 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -99,67 +99,67 @@ Note: You should always set `allow-deprecated-apis=false` to test for changes.
 
 
 [#4829]: https://github.com/ElementsProject/lightning/pull/4829
-[#5085]: https://github.com/ElementsProject/lightning/pull/5085
-[#5103]: https://github.com/ElementsProject/lightning/pull/5103
-[#5086]: https://github.com/ElementsProject/lightning/pull/5086
-[#5103]: https://github.com/ElementsProject/lightning/pull/5103
-[#4985]: https://github.com/ElementsProject/lightning/pull/4985
-[#5078]: https://github.com/ElementsProject/lightning/pull/5078
+[#4829]: https://github.com/ElementsProject/lightning/pull/4829
+[#4864]: https://github.com/ElementsProject/lightning/pull/4864
+[#4890]: https://github.com/ElementsProject/lightning/pull/4890
 [#4896]: https://github.com/ElementsProject/lightning/pull/4896
+[#4897]: https://github.com/ElementsProject/lightning/pull/4897
+[#4900]: https://github.com/ElementsProject/lightning/pull/4900
+[#4902]: https://github.com/ElementsProject/lightning/pull/4902
+[#4902]: https://github.com/ElementsProject/lightning/pull/4902
+[#4902]: https://github.com/ElementsProject/lightning/pull/4902
+[#4906]: https://github.com/ElementsProject/lightning/pull/4906
+[#4908]: https://github.com/ElementsProject/lightning/pull/4908
+[#4911]: https://github.com/ElementsProject/lightning/pull/4911
+[#4913]: https://github.com/ElementsProject/lightning/pull/4913
+[#4921]: https://github.com/ElementsProject/lightning/pull/4921
+[#4925]: https://github.com/ElementsProject/lightning/pull/4925
+[#4929]: https://github.com/ElementsProject/lightning/pull/4929
 [#4931]: https://github.com/ElementsProject/lightning/pull/4931
-[#5104]: https://github.com/ElementsProject/lightning/pull/5104
+[#4945]: https://github.com/ElementsProject/lightning/pull/4945
 [#4984]: https://github.com/ElementsProject/lightning/pull/4984
-[#5078]: https://github.com/ElementsProject/lightning/pull/5078
+[#4985]: https://github.com/ElementsProject/lightning/pull/4985
+[#4985]: https://github.com/ElementsProject/lightning/pull/4985
+[#5004]: https://github.com/ElementsProject/lightning/pull/5004
+[#5010]: https://github.com/ElementsProject/lightning/pull/5010
+[#5010]: https://github.com/ElementsProject/lightning/pull/5010
 [#5013]: https://github.com/ElementsProject/lightning/pull/5013
 [#5013]: https://github.com/ElementsProject/lightning/pull/5013
-[#5086]: https://github.com/ElementsProject/lightning/pull/5086
-[#4906]: https://github.com/ElementsProject/lightning/pull/4906
-[#5122]: https://github.com/ElementsProject/lightning/pull/5122
-[#4925]: https://github.com/ElementsProject/lightning/pull/4925
-[#5146]: https://github.com/ElementsProject/lightning/pull/5146
-[#4911]: https://github.com/ElementsProject/lightning/pull/4911
-[#5103]: https://github.com/ElementsProject/lightning/pull/5103
-[#5051]: https://github.com/ElementsProject/lightning/pull/5051
+[#5016]: https://github.com/ElementsProject/lightning/pull/5016
 [#5019]: https://github.com/ElementsProject/lightning/pull/5019
-[#4829]: https://github.com/ElementsProject/lightning/pull/4829
-[#5121]: https://github.com/ElementsProject/lightning/pull/5121
+[#5043]: https://github.com/ElementsProject/lightning/pull/5043
+[#5047]: https://github.com/ElementsProject/lightning/pull/5047
+[#5047]: https://github.com/ElementsProject/lightning/pull/5047
+[#5047]: https://github.com/ElementsProject/lightning/pull/5047
+[#5051]: https://github.com/ElementsProject/lightning/pull/5051
+[#5052]: https://github.com/ElementsProject/lightning/pull/5052
+[#5058]: https://github.com/ElementsProject/lightning/pull/5058
+[#5078]: https://github.com/ElementsProject/lightning/pull/5078
+[#5078]: https://github.com/ElementsProject/lightning/pull/5078
+[#5078]: https://github.com/ElementsProject/lightning/pull/5078
 [#5081]: https://github.com/ElementsProject/lightning/pull/5081
-[#5130]: https://github.com/ElementsProject/lightning/pull/5130
-[#5120]: https://github.com/ElementsProject/lightning/pull/5120
-[#5010]: https://github.com/ElementsProject/lightning/pull/5010
+[#5085]: https://github.com/ElementsProject/lightning/pull/5085
+[#5085]: https://github.com/ElementsProject/lightning/pull/5085
+[#5086]: https://github.com/ElementsProject/lightning/pull/5086
+[#5086]: https://github.com/ElementsProject/lightning/pull/5086
+[#5090]: https://github.com/ElementsProject/lightning/pull/5090
+[#5101]: https://github.com/ElementsProject/lightning/pull/5101
+[#5103]: https://github.com/ElementsProject/lightning/pull/5103
+[#5103]: https://github.com/ElementsProject/lightning/pull/5103
+[#5103]: https://github.com/ElementsProject/lightning/pull/5103
 [#5104]: https://github.com/ElementsProject/lightning/pull/5104
-[#4900]: https://github.com/ElementsProject/lightning/pull/4900
-[#4921]: https://github.com/ElementsProject/lightning/pull/4921
-[#4897]: https://github.com/ElementsProject/lightning/pull/4897
-[#5016]: https://github.com/ElementsProject/lightning/pull/5016
-[#4929]: https://github.com/ElementsProject/lightning/pull/4929
+[#5104]: https://github.com/ElementsProject/lightning/pull/5104
+[#5120]: https://github.com/ElementsProject/lightning/pull/5120
+[#5121]: https://github.com/ElementsProject/lightning/pull/5121
 [#5121]: https://github.com/ElementsProject/lightning/pull/5121
-[#5058]: https://github.com/ElementsProject/lightning/pull/5058
-[#5052]: https://github.com/ElementsProject/lightning/pull/5052
-[#5010]: https://github.com/ElementsProject/lightning/pull/5010
-[#4902]: https://github.com/ElementsProject/lightning/pull/4902
-[#5047]: https://github.com/ElementsProject/lightning/pull/5047
-[#5004]: https://github.com/ElementsProject/lightning/pull/5004
-[#5146]: https://github.com/ElementsProject/lightning/pull/5146
-[#5043]: https://github.com/ElementsProject/lightning/pull/5043
 [#5122]: https://github.com/ElementsProject/lightning/pull/5122
 [#5122]: https://github.com/ElementsProject/lightning/pull/5122
-[#4985]: https://github.com/ElementsProject/lightning/pull/4985
-[#5047]: https://github.com/ElementsProject/lightning/pull/5047
-[#5090]: https://github.com/ElementsProject/lightning/pull/5090
-[#5078]: https://github.com/ElementsProject/lightning/pull/5078
-[#4864]: https://github.com/ElementsProject/lightning/pull/4864
-[#4945]: https://github.com/ElementsProject/lightning/pull/4945
-[#4902]: https://github.com/ElementsProject/lightning/pull/4902
-[#4902]: https://github.com/ElementsProject/lightning/pull/4902
-[#5047]: https://github.com/ElementsProject/lightning/pull/5047
-[#4913]: https://github.com/ElementsProject/lightning/pull/4913
 [#5122]: https://github.com/ElementsProject/lightning/pull/5122
-[#4890]: https://github.com/ElementsProject/lightning/pull/4890
-[#5101]: https://github.com/ElementsProject/lightning/pull/5101
-[#5085]: https://github.com/ElementsProject/lightning/pull/5085
-[#4908]: https://github.com/ElementsProject/lightning/pull/4908
 [#5122]: https://github.com/ElementsProject/lightning/pull/5122
+[#5122]: https://github.com/ElementsProject/lightning/pull/5122
+[#5130]: https://github.com/ElementsProject/lightning/pull/5130
+[#5146]: https://github.com/ElementsProject/lightning/pull/5146
+[#5146]: https://github.com/ElementsProject/lightning/pull/5146
 [0.11.0]: https://github.com/ElementsProject/lightning/releases/tag/v0.11.0
 
 ## [0.10.2] - 2021-11-03: Bitcoin Dust Consensus Rule
-- 
2.35.1


From a22c059ca4c0cc464206e8d77b9fbe97a0d1b6d4 Mon Sep 17 00:00:00 2001
From: William Casarin <jb55@jb55.com>
Date: Mon, 4 Apr 2022 12:50:25 -0700
Subject: [PATCH clightning 2/2] changelog: remove duplicate pr links

---
 CHANGELOG.md | 21 ---------------------
 1 file changed, 21 deletions(-)

diff --git a/CHANGELOG.md b/CHANGELOG.md
index a3d4d858f..bd06026c7 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -98,7 +98,6 @@ Note: You should always set `allow-deprecated-apis=false` to test for changes.
 
 
 
-[#4829]: https://github.com/ElementsProject/lightning/pull/4829
 [#4829]: https://github.com/ElementsProject/lightning/pull/4829
 [#4864]: https://github.com/ElementsProject/lightning/pull/4864
 [#4890]: https://github.com/ElementsProject/lightning/pull/4890
@@ -106,8 +105,6 @@ Note: You should always set `allow-deprecated-apis=false` to test for changes.
 [#4897]: https://github.com/ElementsProject/lightning/pull/4897
 [#4900]: https://github.com/ElementsProject/lightning/pull/4900
 [#4902]: https://github.com/ElementsProject/lightning/pull/4902
-[#4902]: https://github.com/ElementsProject/lightning/pull/4902
-[#4902]: https://github.com/ElementsProject/lightning/pull/4902
 [#4906]: https://github.com/ElementsProject/lightning/pull/4906
 [#4908]: https://github.com/ElementsProject/lightning/pull/4908
 [#4911]: https://github.com/ElementsProject/lightning/pull/4911
@@ -119,47 +116,29 @@ Note: You should always set `allow-deprecated-apis=false` to test for changes.
 [#4945]: https://github.com/ElementsProject/lightning/pull/4945
 [#4984]: https://github.com/ElementsProject/lightning/pull/4984
 [#4985]: https://github.com/ElementsProject/lightning/pull/4985
-[#4985]: https://github.com/ElementsProject/lightning/pull/4985
 [#5004]: https://github.com/ElementsProject/lightning/pull/5004
 [#5010]: https://github.com/ElementsProject/lightning/pull/5010
-[#5010]: https://github.com/ElementsProject/lightning/pull/5010
-[#5013]: https://github.com/ElementsProject/lightning/pull/5013
 [#5013]: https://github.com/ElementsProject/lightning/pull/5013
 [#5016]: https://github.com/ElementsProject/lightning/pull/5016
 [#5019]: https://github.com/ElementsProject/lightning/pull/5019
 [#5043]: https://github.com/ElementsProject/lightning/pull/5043
 [#5047]: https://github.com/ElementsProject/lightning/pull/5047
-[#5047]: https://github.com/ElementsProject/lightning/pull/5047
-[#5047]: https://github.com/ElementsProject/lightning/pull/5047
 [#5051]: https://github.com/ElementsProject/lightning/pull/5051
 [#5052]: https://github.com/ElementsProject/lightning/pull/5052
 [#5058]: https://github.com/ElementsProject/lightning/pull/5058
 [#5078]: https://github.com/ElementsProject/lightning/pull/5078
-[#5078]: https://github.com/ElementsProject/lightning/pull/5078
-[#5078]: https://github.com/ElementsProject/lightning/pull/5078
 [#5081]: https://github.com/ElementsProject/lightning/pull/5081
 [#5085]: https://github.com/ElementsProject/lightning/pull/5085
-[#5085]: https://github.com/ElementsProject/lightning/pull/5085
-[#5086]: https://github.com/ElementsProject/lightning/pull/5086
 [#5086]: https://github.com/ElementsProject/lightning/pull/5086
 [#5090]: https://github.com/ElementsProject/lightning/pull/5090
 [#5101]: https://github.com/ElementsProject/lightning/pull/5101
 [#5103]: https://github.com/ElementsProject/lightning/pull/5103
-[#5103]: https://github.com/ElementsProject/lightning/pull/5103
-[#5103]: https://github.com/ElementsProject/lightning/pull/5103
-[#5104]: https://github.com/ElementsProject/lightning/pull/5104
 [#5104]: https://github.com/ElementsProject/lightning/pull/5104
 [#5120]: https://github.com/ElementsProject/lightning/pull/5120
 [#5121]: https://github.com/ElementsProject/lightning/pull/5121
-[#5121]: https://github.com/ElementsProject/lightning/pull/5121
-[#5122]: https://github.com/ElementsProject/lightning/pull/5122
-[#5122]: https://github.com/ElementsProject/lightning/pull/5122
-[#5122]: https://github.com/ElementsProject/lightning/pull/5122
-[#5122]: https://github.com/ElementsProject/lightning/pull/5122
 [#5122]: https://github.com/ElementsProject/lightning/pull/5122
 [#5130]: https://github.com/ElementsProject/lightning/pull/5130
 [#5146]: https://github.com/ElementsProject/lightning/pull/5146
-[#5146]: https://github.com/ElementsProject/lightning/pull/5146
 [0.11.0]: https://github.com/ElementsProject/lightning/releases/tag/v0.11.0
 
 ## [0.10.2] - 2021-11-03: Bitcoin Dust Consensus Rule
-- 
2.35.1


```

**Comment by rustyrussell** [comment]
OK, trivial addition to CHANGELOG.md since last-minute merge of #5136 .

**Pull request merged**